### DISCUSSION TITLE
Dynamic content doesn't work in v-tooltip lastest v2.1.3.

### DISCUSSION THREAD

**Discussion by loganbeast** [Akryum/v-tooltip#824] [Q&A]
Description:
Currently I want to make a tooltip integrated with copy clipboard event.
Inital content data = ''
I want when I click to `copy button`, tooltip content should appear as `Copied` (RIGHT AFTER FIRST CLICK). But it doesn't work, it need 2 click to show tooltip content, at first click, the content data was filled but doesn't show up tooltip.

How I want to fix:
It should show up tooltip content at RIGHT FIRST CLICK, then this tooltip hide automatcally after delay time.

Here is reproductive sample.
https://codesandbox.io/s/vue-js-vue-tooltip-example-forked-2r3k5?file=/src/App.vue

[](https://codesandbox.io/s/vue-js-vue-tooltip-example-forked-2r3k5?file=/src/App.vue)

**Comment by Akryum** [comment]
Your logic is wrong, you should setup the tooltip and manually show it:
https://codesandbox.io/s/vue-js-vue-tooltip-example-forked-5h5yg?file=/src/App.vue

**Comment by loganbeast** [comment]
Well, I tried to handle in `manual` way. But supposed I have multiple button in for loop. I have to add showTooltip for each button. I think it's not a good way. So I expected that v-tooltip have this behaviou by itself instead of handling manually.

**Reply by Akryum** [reply]
Or create a reusable component?

**Reply by loganbeast** [reply]
I had solution for this issue. That was my bad.
Thank you so much for supporting.
